example/src/App.js

```diff
@@ -3,9 +3,7 @@ import PlaygroundContainer from './PlaygroundContainer';
 
 function App() {
   return (
-    <body>
-      <PlaygroundContainer title='React JSON Schema Form Builder' />
-    </body>
```

Comment:
Fixes this warning:

![warning](https://user-images.githubusercontent.com/728866/114263553-7d617980-99e6-11eb-801f-db0b3bb64978.png)